lib/language.js

```diff
@@ -6547,6 +6592,7 @@ function init() {
       ,nb: 'Kunngjøring'
       ,fi: 'Tiedoite'
       ,pl: 'Powiadomienie'
+      ,pt: 'Aviso'
```

Comment:
@dougtakeuti that exposed another duplicate here, this file is way too big we need to split it up